examples/pytorch/rgcn/entity_classify_mp.py

```diff
@@ -6,11 +6,9 @@
 * l2norm applied to all weights
 * remove nodes that won't be touched
 """
-import argparse
-import itertools
+import argparse, gc
```

Comment:
there are many modifications in this PR that is irrelevant to the sparse embedding. can we separate these modifications?